test/quantization/test_backward_compatibility.py

```diff
@@ -0,0 +1,126 @@
+        # torch.save(data, input_file)
+        # torch.save(qmodule.state_dict(), state_dict_file)
+        # torch.jit.save(torch.jit.script(qmodule), scripted_module_file)
+        # torch.jit.save(torch.jit.trace(qmodule, data), traced_module_file)
+        # torch.save(qmodule(data), expected_file)
+
+        data = torch.load(input_file)
+        qmodule.load_state_dict(torch.load(state_dict_file))
+        qmodule_scripted = torch.jit.load(scripted_module_file)
+        qmodule_traced = torch.jit.load(traced_module_file)
+
+        expected = torch.load(expected_file)
+        self.assertEqual(qmodule(data), expected, prec=prec)
+        self.assertEqual(qmodule_scripted(data), expected, prec=prec)
+        self.assertEqual(qmodule_traced(data), expected, prec=prec)
+
+    @unittest.skipUnless(
+        'fbgemm' in torch.backends.quantized.supported_engines,
+        " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
+        " with instruction set support avx2 or newer.",
+    )
+    def test_conv2d(self):
+        # quantized conv module
+        qconv = nnq.Conv2d(3, 3, kernel_size=3, stride=1, padding=0, dilation=1,
+                           groups=1, bias=True, padding_mode="zeros")
+        self._test_op(qconv)
+        # TODO: graph mode quantized conv2d module
+
+    @unittest.skipUnless(
+        'fbgemm' in torch.backends.quantized.supported_engines,
+        " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
+        " with instruction set support avx2 or newer.",
+    )
+    def test_conv2d_relu(self):
+        # quantized conv module
+        qconv = nniq.ConvReLU2d(3, 3, kernel_size=3, stride=1, padding=0, dilation=1,
+                                groups=1, bias=True, padding_mode="zeros")
+        self._test_op(qconv)
+        # TODO: graph mode quantized conv2d module
+
+    @unittest.skipUnless(
+        'fbgemm' in torch.backends.quantized.supported_engines,
+        " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
+        " with instruction set support avx2 or newer.",
+    )
+    def test_conv3d(self):
+        # quantized conv module
+        qconv = nnq.Conv3d(3, 3, kernel_size=3, stride=1, padding=0, dilation=1,
+                           groups=1, bias=True, padding_mode="zeros")
+        self._test_op(qconv)
+        # TODO: graph mode quantized conv3d module
+
+    @unittest.skipUnless(
+        'fbgemm' in torch.backends.quantized.supported_engines,
+        " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
+        " with instruction set support avx2 or newer.",
+    )
+    def test_conv3d_relu(self):
+        # quantized conv module
+        qconv = nniq.ConvReLU3d(3, 3, kernel_size=3, stride=1, padding=0, dilation=1,
+                                groups=1, bias=True, padding_mode="zeros")
+        self._test_op(qconv)
+        # TODO: graph mode quantized conv3d module
```

Comment:
For graph mode, you would run the same test as above, except that you wouldnt pass a module and check for state-dict compatibility as the model is scripted.  I think the test would be:
```python
       expected = torch.load(expected_file)
       self.assertEqual(qmodule_scripted(data), expected, prec=prec)
       self.assertEqual(qmodule_traced(data), expected, prec=prec)
```

Reply:
yeah, will add this after the serialization is supported for graph mode